frontend/src/contracts.ts

```diff
@@ -62,8 +62,8 @@ interface Chain {
 }
 
 export function getConfigValue(key: string): any {
-  if(process.env.NODE_ENV === 'development') return '';
-  const env = window.location.href.startsWith('https://test') ? 'test' : 'production';
+  //if(process.env.NODE_ENV === 'development') return '';
+  const env = 'test'; //window.location.href.startsWith('https://test') ? 'test' : 'production';
```

Comment:
This was done probably just for testing, please revert

Reply:
reverted